### PR TITLE
Adding copyright to cert reload files

### PR DESCRIPTION
**Pull request by debjanibnrj**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/main/java/com/amazon/opendistroforelasticsearch/security/ssl/rest/OpenDistroSecuritySSLCertsInfoAction.java [hunk 1]
```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (hardik-k-shah) [inline src/main/java/com/amazon/opendistroforelasticsearch/security/ssl/rest/OpenDistroSecuritySSLCertsInfoAction.java:2]: 2020

**Comment by hardik-k-shah** [comment]
Can you also check throughout repo and update if it is missing?

**Comment by vrozov** [comment]
It will be ok to update header for cert reload files (unit tests also needs to be updated) in this PR and all other files in a separate PR.

**Comment by debjanibnrj** [comment]
@vrozov @hardik-k-shah Thanks for the suggestion. In this Pull request I have only included cert reload file copyright changes. I will make another pull request for the remaining files.

**Review by hardik-k-shah — APPROVED** [review]

**Review by palashhedau — APPROVED** [review]

**Review by vrozov — APPROVED** [review]

**Pull request merged**